File: solutions/TypeScript/[0171_Easy] Excel Sheet Column Number/titleToNumber.ts
function titleToNumber(columnTitle: string): number {
  let sum = 0;
  for (let i = columnTitle.length - 1, j = 0; i >= 0; i--, j++) {
    sum += (columnTitle.charCodeAt(i) - 64) * Math.pow(26, j);
  }
  return sum;
}

File: solutions/Java/[0171_Easy] Excel Sheet Column Number/titleToNumber.java
class Solution {
  public int titleToNumber(String columnTitle) {
    int sum = 0;
    for (int i = columnTitle.length() - 1, j = 0; i >= 0; i--, j++) {
      sum += (columnTitle.charAt(i) - 64) * Math.pow(26, j);
    }
    return sum;
  }
}
